packages/fluentui/react-docs/README.md

```diff
@@ -0,0 +1,84 @@
+
+`getComponentInfo(<component>)` will return an object that can be serialized and then used as need.
+
+The simplest signature of `getComponentInfo` is as follows
+
+```JSX
+import { getComponentInfo } from '@fluentui/react-docs'
+import { SomeComponent } from '.././SomeFileContainingSomeComponent'
+
+const componentInfo = getComponentInfo(SomeComponent)
+```
+
+## Schema Resolver Callback
+
+```js
+schemaResolverCallback(fileInfo, ignoredParentInterfaces, componentInfo);
+```
+
+_react-docs_ allows you to modify the object generated via the schema resolver callback. This callback receives information from _getComponentInfo_ about the component to parse.
+
+| Arguments                           |                                                                                                                                                                                                                       |
+| ----------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `<signature>`                       | (**fileInfo**, **ignoredParentInterfaces**, **componentInfo**)                                                                                                                                                        |
+| fileInfo: `object`                  | absPath: `string` <br> dir: `string` <br> dirname: `string` <br> filename: `string` <br> filenameWithoutExt: `string` <br> info: `docgen.ComponentDoc`                                                                |
+| ignoredParentInterfaces: `string[]` |                                                                                                                                                                                                                       |
+| componentInfo: `object`             | constructorName: `string`<br>Component: `React.Component`<br>displayName: `string`<br>docblock `string`<br>filename `string`<br>filenameWithoutExt `string`<br>props `string[]`<br>repoPath `string`<br>type `string` |
+
+## Errors
+
+### Error: Could not find a component definition in "YourComponent.tsx".
+
+This error occurs when your component file does not have a recognizable React component exported.
+
+Supported component and export formats:
+
+```jsx
+const Component = React.createClass(<def>);
+module.exports = Component
+
+// or
+
+class Component extends React.Component {
+  // ...
+}
+module.exports = Component
+
+// or
+
+const Component = (props) => {
+  // ...
+}
+export default Component
+```
+
+**NOT** supported formats
+
+```jsx
+const Component = props => {
+  // ...
+};
+module.exports = Component;
```

Comment:
Is there any difference between previous one?

Reply:
The previous example uses `( )` while line 71 does not.